1.8/administration/overlay-networks/ip-per-container.md

```diff
@@ -4,30 +4,28 @@ post_title: Configuring IP-per-Container in Overlay Networks
 menu_order: 10
 ---
 
-In Enterprise DC/OS the overlay network feature is enabled by default. The default configuration provides two overlay networks `dcos-1` and `dcos-2`, whose YAML configuration is as follows:
+In DC/OS, the overlay network feature is enabled by default. The default configuration provides an overlay networks `dcos`, whose YAML configuration is as follows:
```

Comment:
How about:

"The overlay network feature is enabled by default in DC/OS. The default configuration provides an overlay network, `dcos`, whose YAML configuration is as follows:"